workflow-ui/core-android/build.gradle.kts

```diff
@@ -40,6 +40,7 @@ dependencies {
   api(Dependencies.Kotlin.Stdlib.jdk6)
 
   implementation(Dependencies.AndroidX.activity)
+  implementation(Dependencies.AndroidX.appcompat)
```

Comment:
Uh-oh. Did we move this out of core to keep @Armaxis happy? Is that still a concern?

I think I can avoid it, but it'll be a bit tedious.

Reply:
No longer a concern, with the amount of things we need these day, appcompat is imminent. 
Bring it back. Sorry for bothering you remove it in first place :/

Reply:
Yay! Glad I asked.